sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -929,81 +929,95 @@ def sdc_pandas_dataframe_drop_impl(df, _func_name, args, columns):
     return sdc_pandas_dataframe_drop_impl(df, _func_name, args, columns)
 
 
-def df_getitem_str_literal_idx_main_codelines(self, idx):
+def df_getitem_slice_idx_main_codelines(self, idx):
```

Comment:
So... this should work not only for slice, write? For any idx. E.g. for array of booleans (filter)?

Reply:
Yes, it should work only for slice.

Reply:
I avoided such code generation for `StringLiteral` and `Unicode` types.

Reply:
I gonna check it works for any `idx` in separate PR.